src/axis.ts

```diff
@@ -134,37 +222,44 @@ const COMMON_AXIS_PROPERTIES_INDEX: Flag<keyof (VgAxis | Axis)> = {
   bandPosition: 1,
   domain: 1,
   domainColor: 1,
+  domainOpacity: 1,
   domainWidth: 1,
   format: 1,
   grid: 1,
   gridColor: 1,
   gridDash: 1,
   gridOpacity: 1,
+  gridScale: 1,
```

Comment:
gridScale shouldn't be supported in Vega-Lite

Reply:
Good point. How do we remove it, though?